.github/workflows/cfntsci.yml

```
@@ -0,0 +1,42 @@
+---
+name: cfntsCI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  Testing:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checking out
+        uses: actions/checkout@v3
+      - name: Setting up Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          components: clippy, rustfmt
+          override: true
+      - name: Rust cache
+        uses: Swatinem/rust-cache@v1
+      - name: Linting
+        run: cargo clippy --all-targets -- -D warnings
+      - name: Format
+        run: cargo fmt --all --check
+      - name: Building
+        run: cargo build --release
+      - name: Testing
+        run: cargo test -- --nocapture
+  e2e:
```

nit (consistent with other job):
```suggestion
  E2E:
```